src/test/java/com/arpnetworking/kairosdb/performance/FactoryPT.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2018 Inscope Metrics, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.kairosdb.performance;
+
+import com.arpnetworking.kairosdb.HistogramFormingMetrics;
+import com.arpnetworking.metrics.generator.metric.GaussianCountMetricGenerator;
+import com.arpnetworking.metrics.generator.metric.GaussianMetricGenerator;
+import com.arpnetworking.metrics.generator.name.SingleNameGenerator;
+import com.google.gson.JsonObject;
+import org.apache.commons.math3.random.MersenneTwister;
+import org.junit.Test;
+import org.kairosdb.core.DataPoint;
+import org.kairosdb.core.datapoints.DataPointFactory;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+/**
+ * Base class for serialization tests.
+ *
+ * @author Brandon Arp (brandon dot arp at inscopemetrics dot com)
+ */
+public abstract class FactoryPT {
```

Comment:
The naming convention for these has changed. Please update.

Reply:
Also, you should not have an abstract test with concrete tests. This looks like you are parameterizing; there are facilities for this in JUnit.